DeepStorage/Patch_Notify_LostThing.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using HarmonyLib;
+using RimWorld;
+using Verse;
+
+namespace LWM.DeepStorage
+{
+    [StaticConstructorOnStartup]
+    public static class Patch_Notify_LostThing
+    {
+        static Patch_Notify_LostThing()
+        {
+            MethodInfo original = typeof(Building_Storage).GetMethod(nameof(Building_Storage.Notify_LostThing));
```

Comment:
In my experiments, this flat out crashes RW on attempt to patch, because the compiled IL code for the function is one operation: OpCodes.Ret.  There isn't room to set up the Harmony detour without overwriting memory.

Does this patch actually work for you?

Reply:
It works on my end.
If you put a Log.Message() in here, it won't get trigger until something is taking out and putting into the storage.